4-WebApp-your-API/4-1-MyOrg/README.md

```diff
@@ -14,28 +14,37 @@ products:
 
 # How to secure an ASP.NET Core Web API with the Microsoft identity platform
 
-[![Build status](https://identitydivision.visualstudio.com/IDDP/_apis/build/status/AAD%20Samples/.NET%20client%20samples/ASP.NET%20Core%20Web%20App%20tutorial)](https://identitydivision.visualstudio.com/IDDP/_build/latest?definitionId=819)
+[![Build status](https://identitydivision.visualstudio.com/IDDP/_apis/build/status/AAD%20Samples/.NET%20client%20samples/ASP.NET%20Core%20Web%20App%20tutorial)](https://identitydivision.visualstudio.com/IDDP/_build/latest?definitionId=XXX)
 
 Table Of Contents
 
+* [Overview](#Overview)
 * [Scenario](#Scenario)
+* [Contents](#Contents)
 * [Prerequisites](#Prerequisites)
 * [Setup the sample](#Setup-the-sample)
-* [Troubleshooting](#Troubleshooting)
+* [Explore the sample](#Explore-the-sample)
+* [We'd love your feedback!](#We'd-love-your-feedback!)
 * [Using the sample](#Using-the-sample)
+* [Troubleshooting](#Troubleshooting)
 * [About the code](#About-the-code)
 * [How the code was created](#How-the-code-was-created)
 * [How to deploy this sample to Azure](#How-to-deploy-this-sample-to-Azure)
```

Comment:
>[How to deploy this sample to Azure [](http://example.com/codeflow?start=2&length=35)

Please test and confirm if the steps are upto date. update code gen if changes are found